docs/notifications/notification-services/slack.md

```diff
@@ -1,112 +1,111 @@
-
-![notification](../../../assets/notifications/services/slack/notification-light-2.png#only-light)
-![notification](../../../assets/notifications/services/slack/notification-dark-2.png#only-dark)
+![settings](../../assets/notifications/services/slack/settings-light-1.png#only-light)
+![settings](../../assets/notifications/services/slack/settings-dark-1.png#only-dark)
 
 ## Add Slack Notification
 
 **Step 1:** Click on the **Add Notifications** button located in the top right corner.
 
-![add-notification](../../../assets/notifications/services/slack/add-notification-light-3.png#only-light)
-![add-notification](../../../assets/notifications/services/slack/add-notification-dark-3.png#only-dark)
+![add-notification](../../assets/notifications/services/slack/add-notification-light-3.png#only-light)
+![add-notification](../../assets/notifications/services/slack/add-notification-dark-3.png#only-dark)
 
 A modal window **Add Notification Rule** will appear providing you with fields to set notification rules.
 
-![modal-window](../../../assets/notifications/services/slack/modal-window-light-4.png#only-light)
-![modal-window](../../../assets/notifications/services/slack/modal-window-dark-4.png#only-dark)
+![modal-window](../../assets/notifications/services/slack/modal-window-light-4.png#only-light)
+![modal-window](../../assets/notifications/services/slack/modal-window-dark-4.png#only-dark)
 
 **Step 2:** Enter the following details to add the notification rule.
 
 **1. Name:** Enter a specific and descriptive title to your notification rule to easily identify its purpose.
 
 **2. Description:** Provide a brief description of what the notification rule does or when it should trigger.
 
+![enter-details](../../assets/notifications/services/slack/enter-details-light-55.png#only-light)
+![enter-details](../../assets/notifications/services/slack/enter-details-dark-55.png#only-dark)
+
 **3. Trigger When**: Select the event or condition from the dropdown menu that will trigger the notification. Below is the list of available events you can choose from:
 
-- **Operation Completion:** This type of notification is triggered whenever an operation, such as a catalog, profile, or scan, is completed on a source datastore. Upon completion, teams are promptly notified through in-app messages and, if configured, via external notification channels such as email, Slack, Microsoft Teams, and others. For example, the team is notified whenever the catalog operation is completed, helping them proceed with the profile operation on the datastore. 
+- **Operation Completion:** This type of notification is triggered whenever an operation, such as a catalog, profile, or scan, is completed on a source datastore. Upon completion, teams are promptly notified through in-app notifications and, the Microsoft Teams channel. For example, the team is notified whenever the catalog operation is completed, helping them proceed with the profile operation on the datastore. 
```

Comment:
IDK if should we keep `Microsoft Teams channel`. Because we remove the examples`if configured, via external notification channels such as email, Slack,  [...] , and others`.

Maybe or give examples like we had or remove all examples and leave in-app notifications.